crates/tests/src/e2e/ledger_tests.rs

```diff
@@ -2322,3 +2322,150 @@ fn rollback() -> Result<()> {
 
     Ok(())
 }
+
```

Comment:
Can we just add another line in the comment here to specify that we need this test to verify the correctness of the proofs generation/verification process and that this cannot be achieved in the integration tests because of the mocks? Just saying cause I can see ourselves forgetting about this thing an being tempted to remove this test later down the road 😅

Reply:
sure can!

Reply:
added [55406a9](https://github.com/anoma/namada/pull/3365/commits/55406a9edee320c9e35a7f2e90366c1d5791e451)